test/ctia/http/generative/atom_store_spec.clj

```diff
@@ -38,7 +38,10 @@
   specs/spec-judgement-routes)
 
 (defspec spec-sighting-routes-atom-store
-  specs/spec-ttp-routes)
+  specs/spec-sighting-routes)
 
 (defspec spec-ttp-routes-atom-store
   specs/spec-ttp-routes)
+
+(defspec spec-bundle-routes-atom-store 50
```

Comment:
I would like us to limit complexity in the generators rather than override the number of tests.  Seeing 50 here is a smell, I think.  I believe that if we are specifying the number of tests, then there is something wrong with our generators.